docs/blueprints/EP035.rst

```diff
@@ -0,0 +1,100 @@
+:EP: 35
+:Title: Kytos stats with OF table stats
+:Authors:
+    - Gretel Liz De la Peña <gdelapea AT fiu DOT edu>
+    - Italo Valcy <idasilva AT fiu DOT edu>
+    - Vinicius Arcanjo <vindasil AT fiu DOT edu>
+:Created: 2023-03-13
+:Kytos-Version: 2023.1
+:Status: Draft
+
+***************************************
+EP035 - Kytos stats with OF table stats
+***************************************
+
+Abstract
+========
+
+This blueprint is to support table statistics and concentrate the statistics into a single NApp.
+Actions will involve ``flow_stats`` and ``of_core``.
+
+Motivation
+==========
+
+Currently ``flow_stats`` is responsible to deal with flow statistics, which are exposed in endpoints:
+
+- ``GET v1/flow/stats`` to get the flows stats by dpid.
+
+- ``GET v1/packet_count/<flow_id>`` for packet count of a specific flow.
+
+- ``GET v1/bytes_count/<flow_id>`` for bytes count of a specific flow.
+
+- ``GET v1/packet_count/per_flow/<dpid>`` for packet count per flow.
+
+- ``GET v1/bytes_count/per_flow/<dpid>`` for bytes count per flow.
+
+However, OF table stats are not supported. 
+
+The goal of this blueprint is to evolve ``flow_stats`` into ``kytos_stats`` for kytos statistics, including OF table stats support.
+
+Specification
+=============
+
+Information about tables is requested with the ``OFPMP_TABLE`` type of the ``OFPT_MULTIPART_REQUEST`` message. 
+The request does not contain any data in the body.
+The body of the reply consists of an array of struct ``ofp_table_stats``, which contains:
+- `table_id`: Identifier of table.
+- `active_count`: Number of active entries.
+- `lookup_count`: Number of packets looked up in table.
+- `matched_count`: Number of packets that hit table. 
+
+Proposed Changes
+----------------
+
+- Rename ``flow_stats`` to ``kytos_stats``. Adapt the zabbix wrapper to send requests to the correct endpoint.
+- New event from ``of_core``.
+- Update ``kytos_stats`` to suport OF table stats.
+- Update Zabbix Wrapper to print table stats.
```

Comment:
Just a heads up that FIU VIP students are also trying to help [with e2e tests for `flow_stats`]( https://github.com/kytos-ng/kytos-end-to-end-tests/issues/203), we'll also need a task to update the e2e routes if they manage to successfully add a test suite for `flow_stats`. Maybe creating/mapping a new issue will make things easier, since they can still iterate on it, and although it's a breaking change, it's just a matter of renaming/refactoring API routes.